locale/Makefile.def

```diff
@@ -0,0 +1,67 @@
+#
+# Definition of common Makefile tasks for PO merging and MO generation.
+#
+# make all-mo (default) - generate MO files
+# make check - check translations using translate-tool
+# make tx-update - download and merge translations from Transifex
+# make clean - clean everything
+#
+
+POTFILE = $(DOMAIN).pot
+MOFILE = $(DOMAIN).mo
+POFILES = $(shell find . -name '$(DOMAIN).po')
+MOFILES = $(patsubst %.po,%.mo,$(POFILES))
+POXFILES = $(patsubst %.po,%.pox,$(POFILES))
+EDITFILES = $(patsubst %.po,%.edit.po,$(POFILES))
+TIMESTAMPFILES = $(patsubst %.po,%.po.time_stamp,$(POFILES))
+
```

Comment:
How about adding help target?
```bash
.PHONY: help

help: ## This help message
	@echo -e "$$(grep -hE '^\S+:.*##' $(MAKEFILE_LIST) | sed -e 's/:.*##\s*/:/' -e 's/^\(.\+\):\(.*\)/\\x1b[36m\1\\x1b[m:\2/' | column -c2 -t -s :)"
```
Then commenting the target with double hash like: `clean: ## Clean everything` would produce:

```
$ make help
help   This help message
...
check  Check translations using translate-tool
clean  Clean everything
```
Seems useful. I've seen it adopted by some other projects

Reply:
Nice, I added that.